docs/source/markdown/podman-images.1.md

```diff
@@ -29,11 +29,11 @@ Filter output based on conditions provided
 
   Filters:
 
-  **after==TIMESTRING**
-    Filter on images created after the given time.Time.
+  **after=IMAGE**, **since=IMAGE**
```

Comment:
Should we stop documenting "after"  and just go with since?

Reply:
I don't have a feeling for who uses this, but I'm OK with it. Shall I update & push?

Reply:
I'd be fine with dropping "after" from the docs but keeping it as a hidden option in the code.

Reply:
Done. Fixed, rebased, pushed. I also changed the order of the options in the `case` statement, putting "since" first to make it clearer to maintainers that it's the preferred option.